pennylane/qchem/convert.py

```diff
@@ -602,12 +604,28 @@ def import_state(solver, tol=1e-15):
         wf_dict = _rccsd_state(solver, tol=tol)
     elif "UCCSD" in method:
         wf_dict = _uccsd_state(solver, tol=tol)
+    elif "tuple" in method:
+        if isinstance(solver[0][0], np.str_):
+            wf_dict = _shci_state(solver, tol=tol)
+        elif isinstance(solver[0][0][0], type(np.array([0]).numpy()[0])):
+            wf_dict = _dmrg_state(solver, tol=tol)
+        else:
+            raise ValueError(
+                "for tuple input, the supported objects are"
+                " tuple(array[str], array[float]) for SHCI calculations with Dice library and "
+                "tuple(array[int], array[float]) for DMRG calculations with the Block2 library."
+            )
     else:
         raise ValueError(
             "The supported objects are RCISD, UCISD, RCCSD, and UCCSD for restricted and"
-            " unrestricted configuration interaction and coupled cluster calculations."
+            " unrestricted configuration interaction and coupled cluster calculations, and"
+            " tuple(array[str], array[float]) for SHCI calculations with Dice library and "
+            "tuple(array[int], array[float]) for DMRG calculations with the Block2 library."
```

Comment:
Please modify these after using list for determinants.

Reply:
Done.